dist/index.html

```diff
@@ -5,88 +5,101 @@
     <title>Seek-A-Design Playground</title>
     <link rel="stylesheet" href="style.css">
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
-    <script src="../src/index.js"></script>
     <script src="https://kit.fontawesome.com/9a26908d19.js"></script>
+    <script src="../src/index.js"></script>
   </head>
   <body>
     <div class="container-parent">
       <div class="container-header">
         <div class="container-logo">
-            <img src="https://i.ibb.co/47tS3wH/seeklogo.png" alt="seeklogo" border="0" id="mainlogo">
+          <img src="https://i.ibb.co/47tS3wH/seeklogo.png" alt="seeklogo" id="mainlogo">
         </div>
         <div class="container-navSection">
           <div class="container-nav">
-                <div class="menu"> <a href="home.asp" class="fas fa-home">Home</a> </div>
-                <div class="menu"> <a href="projects.asp" class="fas fa-desktop">Projects</a> </div>
-                <div class="menu"> <a href="about.asp" class="fas fa-user">About_Me</a> </div>
-                <div class="menu"> <a href="contact.asp" class="fas fa-envelope">Contact</a> </div>
+
+            <div class="menu">
+              <span class="fas fa-home"></span><p>Home</p>
+            </div>
+            <div class="menu">
+              <span class="fas fa-desktop"></span><p>Projects</p>
+            </div>
+            <div class="menu">
+              <span class="fas fa-user"></span><p>About Me</p>
+            </div>
+            <div class="menu">
+                <span class="fas fa-envelope"></span><p>Contact</p>
+            </div>
+
           </div>
           <div class="container-moto">
-            <h2>Custom Designs · Marketing · Branding · And More </h2>
+            <h2>Custom Designs · Marketing · Branding · And More</h2>
           </div>
         </div>
       </div>
       <div class="container-body">
         <div class="content">
           <div class="content-left">
-            <span class ="ownerContact" style="color: #336D61; font-size: 3em; font-family: AlexandriaFLF;">
+            <span class="ownerContact">
               CONTACT
             </span>
-            </h2>
-              <hr>
-            <p style="text-align: center;">
-              <span class="ownerName">
-                LYANDRA SICA<br/>
-              </span>
+            <!-- TODO: Replace with static horizontal rule asset when delivered by designer. -->
+            <hr>
+              <div class="ownerTitle">
+                <span class="ownerName">
+                  LYANDRA SICA<br/>
+                </span>
+                Creative Designer<br/>
+              </div>
+              <p class="text-center">
```

Comment:
I think maybe we should chat about the best times to use `paragraph`, `span`, and `div` tags.

Here you have a paragraphs containing many spans and divs. The general pattern I've observed is that:

- `<div>` tags are used as general containers. They can contain `<p>`s, `<span>`s, and just about anything else.
- `<p>` tags can contain `<span>` tags, but only when a subsection of the text in the paragraph needs special styling. `<p>` tags never contain `<div>` tags.
- `<span>` tags don't usually contain anything other than text, and are generally used for styling a subsection of text in-line. The only tags that make sense as children a `<span>` tags are `<a>`, `<img>`, `<sup>`, etc.

Check out [this article](https://stackoverflow.com/questions/11314668/what-elements-can-a-span-tag-contain-in-html5) from Stack Overflow!